view/frontend/web/instantsearch.js

```diff
@@ -639,16 +639,18 @@ define(
             var isStarted = false;
 
             // Capture active redirect URL with IS facet params for add to cart from PLP
-            search.on('render', () => {
-               const cartForms = document.querySelectorAll('[data-role="tocart-form"]');
-               cartForms.forEach((form, i) => {
-                   const ts = Date.now();
```

Comment:
`ts` constant is not actually being used anywhere

Reply:
Good catch - thanks! We'll be sure to remove this.